src/PKSim.UI/UIConstants.cs

```diff
@@ -33,7 +33,7 @@ public static class Size
          public static readonly int PROTOCOL_VIEW_WIDTH = ScaleForScreenDPI(700);
          public static readonly int PROTOCOL_VIEW_HEIGHT = ScaleForScreenDPI(760);
 
-         public static readonly int EXPRESSION_PROFILE_LOCALIZATION_HEIGHT = ScaleForScreenDPI(80);
+         public static readonly int EXPRESSION_PROFILE_LOCALIZATION_HEIGHT = ScaleForScreenDPI(85);
```

Comment:
@PavelBal This is for your localization view issue. Can you try and let me know if this fixes #2275